eul/de.lproj/Localizable.strings

```diff
@@ -135,4 +135,4 @@
 "ui.network" = "Netzwerk";
 "ui.menu_view" = "Menü Ansicht";
 "ui.empty" = "Empty";
-"ui.status_bar_view" = "Status Bar View";
+"ui.status_bar_view" = "Status Bar Ansicht";
```

Comment:
@akeschmidi can you have a review of the translation?

Reply:
that's correct